packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal-header.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { DemoLight, DemoDark } from '../../../public/images/index';
+import classNames from 'classnames';
+import { Badge, GradientBackground, Icon, Text, Watermark } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { getWalletCurrencyIcon } from 'Constants/utils';
+import { getCashierWalletModalBackgrounds } from 'Constants/wallet-backgrounds';
+
+type TCashierWalletModalHeaderProps = {
+    balance?: string | number;
+    currency: string;
+    is_dark: boolean;
+    is_demo: boolean;
+    is_mobile: boolean;
+    landing_company_shortcode: 'svg' | 'malta';
```

Comment:
Maybe better to use `shortcode` instead of `landing_company_shortcode`? You can ignore it